pkg/engine/loadbalancers.go

```diff
@@ -259,26 +259,3 @@ func CreateMasterInternalLoadBalancer(cs *api.ContainerService) LoadBalancerARM
 
 	return loadBalancerARM
 }
-
-func getUDPLoadBalancingRule() network.LoadBalancingRule {
```

Comment:
Is this dead code that my PR introduced?

Reply:
Yes, I think so. But the specific error has been hidden by this problem.

Reply:
Verified. I recollect I scaffolded out this convenience so I could re-use it, then realized the re-use case was not as clean as I thought, and so abandonded the re-usable function path. Thanks for cleaning up the func for me! :)